spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/SpannerConverter.java

```diff
@@ -75,15 +75,32 @@
 	 * @param targetType the desired target type
 	 * @return
 	 */
-	boolean canConvert(Class sourceType, Class targetType);
+	boolean canWriteConvert(Class sourceType, Class targetType);
 
 	/**
 	 * Converts the given object to the desired target type.
 	 * @param source the object to convert
 	 * @param targetType the desired target type
 	 * @return the converted object.
 	 */
-	Object convert(Object source, Class targetType);
+	Object readConvert(Object source, Class targetType);
```

Comment:
Not an error any thing, but it's interesting that this function is greyed out in my intellij and it's not used anywhere.

Also, it seems like the `canReadConvert` function is only used in a test, too. So perhaps these two functions aren't needed in this interface?

Reply:
sure, we can remove this delegation to the specific converters, I'm up for that - we have to expose the right converters though for - which is already kind of happening. I'm going to experiment with it in the next commit.